src/content/en/updates/2019/09/fresher-sw.md

```diff
@@ -93,6 +104,40 @@ if ('serviceWorker' in navigator) {
 }
 ```
 
+## Checks for updates to imported scripts
+
+Prior to Chrome 78, any service worker script loaded via
+[`importScripts()`](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts)
+would be retrieved only once (checking first against the HTTP cache, or via the
+network, depending on the `updateViaCache` configuration). After that initial
+retrieval, it would be cached internally by the browser, and never re-fetched.
+
+The only way to force an already installed service worker to pick up changes to
+an imported script was to change the script's URL, either usually by adding in a
+[semver value](https://semver.org/) (e.g.
+`importScripts('http://example.com/v1.1.0/index.js')`) or by including a hash of
+the contents (e.g. `importScripts('https://example.com/index.abcd1234.js')`). A
+side-effect of changing the imported URL is that the top-level service worker
+script's contents change, which in turn triggers the
+[service worker update flow](/web/fundamentals/primers/service-workers/lifecycle#updates).
```

Comment:
I think you could also use max-age=0 for the imported script and change the main script contents (may be too detailed though)

Reply:
Yeah, I think I can leave that detail out.